### DISCUSSION TITLE
The copilot extension says Copilot activation failed

### DISCUSSION THREAD

**Discussion by TheDiamondOG** [community/community#9572] [Copilot Conversations] [answered]
I don't know if I have to wait to get to use copilot because it is in beta or if it is a bug
Console:
[INFO] [default] [2022-01-03T02:34:06.868Z] Experimentation service initialized
[INFO] [auth] [2022-01-03T02:34:07.164Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-01-03T02:34:07.165Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
[INFO] [default] [2022-01-03T02:34:07.306Z] Stringified features: {"copilotaa":"false","copilotincludeprompt":"medium"}
[INFO] [default] [2022-01-03T02:34:07.306Z] Feature for copilotaa: false
[INFO] [default] [2022-01-03T02:34:07.306Z] Feature for copilotincludeprompt: medium
[INFO] [default] [2022-01-03T02:34:07.306Z] Feature for copilotset1: undefined
[INFO] [default] [2022-01-03T02:34:07.306Z] Feature for copilotghgh: false
[INFO] [default] [2022-01-03T02:34:07.306Z] Feature for copilotfim: false

**Comment by davecheney** [comment, marked as answer]
Did you receive an email inviting you to the copilot beta. If not, it is likely the reason you are receiving this message is you are on the waitlist.

**Reply by TheDiamondOG** [reply]
Ok I will check that

**Reply by TheDiamondOG** [reply]
I did not get an email yet I will update if that fixes it or not

Edit: I think it is that I have not got an email yet because it says User not authorized when I click the button to turn on copilot

**Comment by yanncoinmena** [comment]
smae here, VS on Ubuntu, Invalid copilot token: missing token: 404 
[ERROR]

**Reply by davecheney** [reply]
@yanncoinmena #
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by kenan0122** [reply]
I try failed

**Reply by davecheney** [reply]
Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Comment by yanncoinmena** [comment]
Thanks.... after login in again, it took some time, 1 or 2 minutes before starting to work

**Comment by shlomiandadi** [comment]
Try turning the plug-in off and on.
It helped me.

**Comment by sedghi** [comment]
This happens for me as well. Co-pilot works, but just annoying message
